src/main/java/net/mcreator/ars_technica/common/entity/WhirlEntity.java

```diff
@@ -152,7 +152,7 @@ private void handleWhirlwindEffect() {
 
     @Override
     protected void defineSynchedData() {
-        this.entityData.define(PROCESSOR_TYPE, AllFanProcessingTypes.NONE.toString());
+        this.entityData.define(PROCESSOR_TYPE, new AllFanProcessingTypes.SplashingType().toString());
```

Comment:
```suggestion
        this.entityData.define(PROCESSOR_TYPE, "");
```

We probably need this, from [here](https://github.com/zeroregard/Ars-Technica/blob/1.21.X/src/main/java/com/zeroregard/ars_technica/entity/ArcaneWhirlEntity.java):
```java
        if (PROCESSOR_TYPE.equals(key)) {
            var val = this.entityData.get(PROCESSOR_TYPE);
            if(val.isEmpty()) {
                this.processor = null;
            } else {
                this.processor = AllFanProcessingTypes.parseLegacy(this.entityData.get(PROCESSOR_TYPE));
            }
            initSound();
        }
```